contribute/README.md

```diff
@@ -10,9 +10,13 @@ Thanks for enhancing OpenFisca anyway!
 
 ## Why contribute to OpenFisca?
 
-OpenFisca is a project being developed under the GPLv3 license or later.
+OpenFisca is a project being developed under the AGPL-3.0 license.
 The source code is freely available and modifiable.
 
+We ask for improvements and evolution of OpenFisca to be shared back, for it allows the community to grow through improvements to the _core engine_ and updates to the _modelled tax-benefit systems_ (country packages, extensions, reforms, etc.).
+
+Although, as long as the software you build with OpenFisca can't be considered an improvement or an evolution of OpenFisca, we won't consider it a form of derivative work. If the latter holds true, we won't ever ask you to share your software's source code, whether you're using the Python API, the Web API or both.
```

Comment:
Wouldn't `However` work better than `Although` ?
> Proposition d'ajout d'une checklist: 
- You are creating a new country package --> Share it back
- You are using a modified version of openfisca-core --> Share it back
- You are working on an extension for an existing coutry-package --> Share it back
- You are working on a reform --> share it back
- you are working on an app that uses a openfisca country package to run simulations --> you don't need to share your code (but we encourage it :) )

Reply:
I'm not really sure it is a good idea, I just want to make an exception here.